hs_tools_resource/models.py

```diff
@@ -534,12 +534,12 @@ class ToolMetaData(CoreMetaData):
     _homepage_url = GenericRelation(AppHomePageUrl)
 
     approved = models.BooleanField(default=False)
-    testing_protocol_url = GenericRelation(TestingProtocolUrl)
-    help_page_url = GenericRelation(HelpPageUrl)
-    source_code_url = GenericRelation(SourceCodeUrl)
-    issues_page_url = GenericRelation(IssuesPageUrl)
-    mailing_list_url = GenericRelation(MailingListUrl)
-    roadmap = GenericRelation(Roadmap)
+    _testing_protocol_url = GenericRelation(TestingProtocolUrl)
+    _help_page_url = GenericRelation(HelpPageUrl)
+    _source_code_url = GenericRelation(SourceCodeUrl)
+    _issues_page_url = GenericRelation(IssuesPageUrl)
+    _mailing_list_url = GenericRelation(MailingListUrl)
+    _roadmap = GenericRelation(Roadmap)
```

Comment:
what is the reason to rename these fields to pre-append a underscore? Would this create confusions when doing db filtering?

Reply:
Now I know after reviewing further a bit - you added properties with the same names. Perhaps appending underscore rather than pre-appending it would work better since db filtering uses double underscore syntax. Just a suggestion for your consideration, though.

Reply:
I renamed these so they're consistent with the other fields.  Can't say I'm a fan of using generic relation for these fields, but thought the least I could do is make it consistent in how it uses generic relations.